contracts/trusttoken/mocks/MockTrustToken.sol

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.6.10;
+
+import {TrustToken} from "../TrustToken.sol";
+import {SafeMath} from "@openzeppelin/contracts/math/SafeMath.sol";
+
+/**
+ * @title MockTrustToken
+ * @dev The TrustToken contract is a claimable contract where the
+ * owner can only mint or transfer ownership. TrustTokens use 8 decimals
+ * in order to prevent rewards from getting stuck in the remainder on division.
+ * Tolerates dilution to slash stake and accept rewards.
+ */
+// prettier-ignore
+contract MockTrustToken is TrustToken {
+    using SafeMath for uint256;
+
+    // last time faucet was called
+    bytes32 private constant stampPosition = 0x2f28007de92226a28af88f681cdc4b6f1674637db295a37f849bd7dee6c8254c; // keccak256(abi.encode(truefi.mock));
+    uint256 constant MAX_FAUCET = 1000000000000;
+    uint256 constant DURATION = 1 minutes;
```

Comment:
Do we want to have global cooldown on calling faucet? Somebody could easily ddos it if they want to